### PR TITLE
api validations on model test passing

### PR DESCRIPTION
**Pull request by Melaza6**

### CODE REVIEW

File: spec/models/cat_spec.rb [hunk 1]
```diff
@@ -1,5 +1,22 @@
 require 'rails_helper'
 
 RSpec.describe Cat, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "should validate all inputs for adding a cat" do
```
Comment (elysemontano) [inline spec/models/cat_spec.rb]: When writing model specs, it is best practice to separate your tests into their own it blocks.

**Review by elysemontano — COMMENTED** [review]
Left a comment to modify on your model spec.

File: spec/models/cat_spec.rb (hunk 1 above)
Comment (Melaza6) [reply spec/models/cat_spec.rb]: on it

Comment (Melaza6) [reply spec/models/cat_spec.rb]: all model specs, with separate test into own it blocks completed and passing.

**Review by elysemontano — APPROVED** [review]
Approving however please clean up your indentation in your spec files before merging.

**Pull request merged**